src/Actions/DeleteMultisiteEntry.php

```diff
@@ -50,4 +56,17 @@ public function run($items, $values)
 
         $items->each->delete();
     }
+
+    private function canChangeBehaviour(): bool
+    {
+        if (! Site::multiEnabled()) {
+            return true;
+        }
+
+        return $this->items->every(function ($entry) {
+            return $entry->isRoot()
+                ? $entry->descendants()->every(fn ($descendant) => User::current()->can("access {$descendant->site()->handle()} site"))
+                : $entry->ancestors()->every(fn ($ancestor) => User::current()->can("access {$ancestor->site()->handle()} site"));
```

Comment:
Why are you doing something different based on whether it's a root entry or not?

I don't think it should ever check ancestors, unless I'm missing something here.

Reply:
> Why are you doing something different based on whether it's a root entry or not?

That's a good question... I can't remember why I added a check for them. I don't think we need it.